integration-test/src/test/java/org/apache/iotdb/db/it/query/IoTDBEncryptionValueQueryIT.java

```diff
@@ -0,0 +1,677 @@
+        "insert into root.ln.wf02.wt02(timestamp,status) values(1509465840000,false)",
+        "insert into root.ln.wf02.wt02(timestamp,status) values(1509465900000,false)",
+        "insert into root.ln.wf02.wt02(timestamp,status) values(1509465960000,false)",
+        "insert into root.ln.wf02.wt02(timestamp,status) values(1509466020000,false)",
+        "insert into root.ln.wf02.wt02(timestamp,status) values(1509466080000,false)",
+        "insert into root.ln.wf02.wt02(timestamp,status) values(1509466140000,false)",
+        "CREATE DATABASE root.sgcc",
+        "create timeseries root.sgcc.wf03.wt01.status with datatype=BOOLEAN,encoding=PLAIN",
+        "insert into root.sgcc.wf03.wt01(timestamp,status) values(1509465600000,true)",
+        "insert into root.sgcc.wf03.wt01(timestamp,status) values(1509465660000,true)",
+        "insert into root.sgcc.wf03.wt01(timestamp,status) values(1509465720000,false)",
+        "insert into root.sgcc.wf03.wt01(timestamp,status) values(1509465780000,false)",
+        "insert into root.sgcc.wf03.wt01(timestamp,status) values(1509465840000,false)",
+        "insert into root.sgcc.wf03.wt01(timestamp,status) values(1509465900000,false)",
+        "insert into root.sgcc.wf03.wt01(timestamp,status) values(1509465960000,false)",
+        "insert into root.sgcc.wf03.wt01(timestamp,status) values(1509466020000,false)",
+        "insert into root.sgcc.wf03.wt01(timestamp,status) values(1509466080000,false)",
+        "insert into root.sgcc.wf03.wt01(timestamp,status) values(1509466140000,false)",
+        "create timeseries root.sgcc.wf03.wt01.temperature with datatype=FLOAT,encoding=RLE",
+        "insert into root.sgcc.wf03.wt01(timestamp,temperature) values(1509465600000,25.957603)",
+        "insert into root.sgcc.wf03.wt01(timestamp,temperature) values(1509465660000,24.359503)",
+        "insert into root.sgcc.wf03.wt01(timestamp,temperature) values(1509465720000,20.092794)",
+        "insert into root.sgcc.wf03.wt01(timestamp,temperature) values(1509465780000,20.182663)",
+        "insert into root.sgcc.wf03.wt01(timestamp,temperature) values(1509465840000,21.125198)",
+        "insert into root.sgcc.wf03.wt01(timestamp,temperature) values(1509465900000,22.720892)",
+        "insert into root.sgcc.wf03.wt01(timestamp,temperature) values(1509465960000,20.71)",
+        "insert into root.sgcc.wf03.wt01(timestamp,temperature) values(1509466020000,21.451046)",
+        "insert into root.sgcc.wf03.wt01(timestamp,temperature) values(1509466080000,22.57987)",
+        "insert into root.sgcc.wf03.wt01(timestamp,temperature) values(1509466140000,20.98177)",
+      };
+
+  @BeforeClass
+  public static void setUp() throws Exception {
+    TSFileDescriptor.getInstance().getConfig().setEncryptFlag("true");
+    TSFileDescriptor.getInstance().getConfig().setEncryptType("AES128");
+    TSFileDescriptor.getInstance().getConfig().setEncryptKey("mkmkmkmkmkmkmkmk");
```

Comment:
This will not be effective for the testing environment because the servers are started in another process.
 Refer to `EnvFactory.getEnv().getConfig()` to see how to change the config in IT.